### DISCUSSION TITLE
Error on realtime service in react native

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#4893] [Q&A] [answered]
```

useEffect(() => {
    pb.collection('connected').subscribe('*', function (e) {
        console.log(e.action);
        console.log(e.record);
    })

  return () => {
    pb.collection('connected').unsubscribe('*');
  }
}, [])
```

Error:Property 'EventSource' doesnt exist

How to use realtime service in react native?

**Comment by ganigeorgiev** [comment, marked as answer]
This was answered several times. Please search for the other similar discussions.

In general you need to register `EventSource` polyfill, ex. `react-native-sse`:

```js
import eventsource from "react-native-sse";

global.EventSource = eventsource;
```

Additionally, if you are testing on Android you may have to disable Flipper. See the links in https://github.com/pocketbase/pocketbase/discussions/2484#discussioncomment-6118523
